src/node_platform.h

```diff
@@ -66,9 +66,9 @@ class PerIsolatePlatformData :
   void ref();
   int unref();
 
-  // Returns true iff work was dispatched or executed.
-  // New tasks that are posted during flushing of the queue are postponed until
-  // the next flushing.
+  // Returns true iff work was dispatched or executed. New tasks that are
```

Comment:
Nit: s/iff/if.

Reply:
@apapirovski I guess these were intentional?

Reply:
Yeah but I know the meaning and I didn't even think of it until you mentioned it... 😆

Reply:
I guess what I'm saying: I don't think this is the place for `iff`. It doesn't make it clearer.